homeassistant/components/notify/pushbullet.py

```diff
@@ -150,14 +152,30 @@ def _push_data(self, message, title, data, pusher, tname=None):
                     if filedata.get('file_type') == 'application/x-empty':
                         _LOGGER.error("Can not send an empty file")
                         return
-                    pusher.push_file(title=title, body=message, **filedata)
+                    if tname:
+                        pusher.push_file(title=title, body=message,
+                                         email=tname, **filedata)
+                    else:
+                        pusher.push_file(title=title, body=message, **filedata)
             elif file_url:
                 if not file_url.startswith('http'):
                     _LOGGER.error("URL should start with http or https")
                     return
-                pusher.push_file(title=title, body=message, file_name=file_url,
-                                 file_url=file_url,
-                                 file_type=mimetypes.guess_type(file_url)[0])
+                if tname:
+                    pusher.push_file(title=title, body=message, email=tname,
+                                     file_name=file_url, file_url=file_url,
+                                     file_type=(mimetypes
+                                                .guess_type(file_url)[0]))
+                else:
+                    pusher.push_file(title=title, body=message,
+                                     file_name=file_url, file_url=file_url,
+                                     file_type=(mimetypes
+                                                .guess_type(file_url)[0]))
```

Comment:
Would it be possible to replace line 164-173 with;
```
pusher.push_file(title=title, body=message, email=tname,
                           file_name=file_url, file_url=file_url,
                           file_type=(mimetypes
                                          .guess_type(file_url)[0]))
```
Since email=None should be the same as not using the argument?

Reply:
Sure, let me take a poke. I was just extending what was there but looking at it more, it looks like the whole thing can be simplified a great deal.